modules/developers-guide/pages/graphql.adoc

```diff
@@ -1,6 +1,109 @@
+
+You can start with a xref:quickstart:quick_start-graphql.adoc[GraphQL QuickStart] or
+follow the full instructions: xref:graphql-oriented-using.adoc[[GraphQL oriented]] xref:graphql-using.adoc[[CQL oriented]] .
+
+We'll show you you how to:
+
+// FIX THE REST OF THESE LINKS!
+// . xref:generating-authorization-token.adoc[Generate an authorization token]
+. xref:manage-application-tokens.adoc[Get an application token]
+. Creating a keyspace  xref:developers-guide/graphql-oriented-using.html#_creating_a_keyspace[[GraphQL oriented]]  xref:developers-guide/graphql-using.html#_creating_a_keyspace[[CQL oriented]]
+. _Optional_: Creating a user-defined type (UDT)  xref:creating-a-udt-in-your-keyspace-GraphQL-oriented.adoc[[GraphQL oriented]]  xref:creating-a-udt-in-your-keyspace-graphql.adoc[[CQL oriented]]
+. Creating table schema  xref:creating-a-table-in-your-keyspace-using-GraphQL-oriented.adoc[[GraphQL oriented]]  xref:creating-a-table-in-your-keyspace-using-graphql.adoc[[CQL oriented]]
+. _Optional_: Creating table indexes  xref:creating-an-index-for-a-table-GraphQL-oriented.adoc[[GraphQL oriented]]  xref:creating-an-index-for-a-table-graphql.adoc[[CQL oriented]]
+. Deploying schema  xref:deploying-schema-in-your-keyspace-GraphQL-oriented.adoc[[GraphQL oriented]]
+. Undeploying schema  xref:undeploying-schema-in-your-keyspace-GraphQL-oriented.adoc[[GraphQL oriented]]
+. Dropping user-defined types (UDTs), tables or columns  xref:dropping-schema-graphql.adoc[[CQL oriented]]
+. Adding rows to your table  xref:adding-rows-in-your-table-using-GraphQL-oriented.adoc[[GraphQL oriented]]  xref:adding-rows-in-your-table-using-graphql.adoc[[CQL oriented]]
+. Retrieving rows from your table  xref:retrieving-rows-in-your-table-using-GraphQL-oriented.adoc[[GraphQL oriented]]  xref:retrieving-rows-in-your-table-using-graphql.adoc[[CQL oriented]]
+. Updating rows in your table  xref:updating-rows-in-your-table-using-GraphQL-oriented.adoc[[GraphQL oriented]]  xref:updating-rows-in-your-table-using-graphql.adoc[[CQL oriented]]
+. Deleting rows in your table  xref:deleting-rows-in-your-table-using-GraphQL-oriented.adoc[[GraphQL oriented]]  xref:deleting-rows-in-your-table-using-graphql.adoc[[CQL oriented]]
+
+== About the GraphQL API endpoints
+
+There are three Stargate GraphQL API endpoints, one for creating schema in CQL-oriented,
+one for deploying schema in the GraphQL-oriented, and the third for querying a keyspace.
+The URLS are:
+
+Schema::
+`https://localhost:8080/graphql-schema`
+Admin::
+`https://localhost:8080/graphql-admin`
+Querying::
+`https://localhost:8080/graphql/{keyspace_name}`
+
+The schema endpoint is used to create schema in the CQL-oriented GraphQL.
+The admin endpoint is used to deploy schema in the GraphQL-oriented GraphQL.
+The querying endpoint is constructed in the same manner for both GraphQL-oriented and CQL-oriented.
+
+Each request must have a valid xref:manage-application-tokens.adoc[application token]
+and a unique id.
```

Comment:
FYI the requestID is optional. Playing around or getting started it can be omitted but in a production environment it's recommended to help with troubleshooting.

Reply:
fixed